README.md

```diff
@@ -16,9 +16,18 @@ The tensorflow/addons repository, will contain additional functionality fitting
 * The addon is useful for a large number of users (e.g., an implementation used in widely cited paper, or a utility with broad applicability)
 
 
----
+# Developing
 
-<br><br>
-<div align="center">
-    <img src ="https://cdn.pixabay.com/photo/2017/06/16/07/26/under-construction-2408062_640.png" />
-</div>
+## Docker
+**Note:** This docker container is just temporary until we can pull a
+tensorflow/tensorflow:custom-op container that reflects nightly changes.
```

Comment:
@yifeif - This is less than ideal; do we have an ETA on the nightly custom op build?

Reply:
I just pushed the first version of the nightly at tensorflow/tensorflow:nightly-custom-op. Hoping to get the nightly push setup by end of week.

Reply:
Thanks!